homeassistant/components/pi_hole/strings.json

```diff
@@ -0,0 +1,23 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "name": "Name",
+          "api_key": "[%key:common::config_flow::data::api_key%] (Optional)",
```

Comment:
Actually, I'm not sure if I can do this.

Reply:
I don't think the `api_key` one will work since they are being translated.